en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,88 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signup) or use your GitHub account if you have one (see GitHub instructions below).
+3. Click _New Project_ and choose _hello-webpage_
+4. Click on the `Tools` dropdown list (at the bottom left side of the window), then on `Terminal` button to open terminal tab with a prompt like this:
+
+{% filename %}Terminal{% endfilename %}
+```
+app@name-of-your-glitch-project:~
+```
+
+When using Glitch.com as your Cloud IDE, we do not create a virtual environment but we create the following files manually :
```

Comment:
Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.
```suggestion
When using Glitch.com as your Cloud IDE, we do not create a virtual environment but we create the following files manually:
```

Reply:
Applied